src/vmm/src/builder.rs

```diff
@@ -1155,6 +1160,7 @@ pub mod tests {
             uffd: None,
             vcpus_handles: Vec::new(),
             vcpus_exit_evt,
+            seccomp_filters: crate::seccomp_filters::get_empty_filters(),
```

Comment:
nit: even if its just a handful of lines, the new seccomp argument could still be in its own refactoring commit :)

Reply:
Seccomp argument given its own refactoring commit